.github/workflows/main.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v2
+      - name: Install Ansible
+        run: pip install --upgrade git+https://github.com/ansible/ansible.git@stable-2.10
       - name: Install molecule
         run: pip install molecule
```

Comment:
Please make this a single pip install. Running pip multiple times gives no guarantees and is always the wrong thing to do. For example, `pip install molecule` can install some other version of ansible.

Also, a note why we need a git checkout is nice.